lib/const.rb

```diff
@@ -130,6 +130,9 @@
 # If CREW_DISABLE_MVDIR environment variable exists and is equal to 1 use rsync/tar to install files in lieu of crew-mvdir.
 CREW_DISABLE_MVDIR = ENV.fetch('CREW_DISABLE_MVDIR', '0').eql?('1')
 
+# Days between crew prompting to run 'crew update' and doing gem updates.
+CREW_UPDATE_CHECK_INTERVAL = 7
```

Comment:
Shouldn't this be set based on whether an environment variable is set?
```
CREW_UPDATE_CHECK_INTERVAL = ENV.fetch('CREW_UPDATE_CHECK_INTERVAL',  7)
```

Reply:
I was considering that!

Reply:
Done.